hibernate-search-elasticsearch/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,284 @@
+                        <div class="description">Relational mapping and full text search</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="ui container search">
+        <form class="ui form" ng-submit="refresh()">
+            <div class="fluid big left ui action input">
+                <input type="text" placeholder="Search for an author name or a book title. Try searching for aus*, owen or mystery." ng-model="pattern">
+                <button class="ui blue right labeled icon button" ng-click="refresh()"><i class="search icon"></i> Search</button>
+            </div>
+        </form>
+    </div>
+    <div class="ui container authors" style="display:none;">
+        <div class="ui cards">
+            <div class="card" ng-repeat="author in authors">
+                <div class="content">
+                    <div class="header">{{ author.firstName }} {{ author.lastName }}</div>
+                    <div class="meta">
+                        {{ author.books.length }} <i class="book icon"></i>
+                        <button class="ui basic blue icon button mini right floated" ng-click="showBookModal(author)"><i class="plus circle icon"></i> Add book</button>
+                    </div>
+                    <div class="description">
+                        <div ng-repeat="book in author.books">
+                            <div>
+                                {{ book.title }}
+                                <a ng-click="remove('book', book)"><i class="trash alternate outline right floated icon"></i></a>
+                            </div>
+                            <div class="ui divider"></div>
+                        </div>
+                    </div>
+                    <div class="extra content">
+                        <div class="ui two buttons">
+                            <button class="ui basic green button" ng-click="showEditAuthorModal(author)"><i class="pencil alternate icon"></i> Edit</button>
+                            <button class="ui basic red button" ng-click="remove('author', author)"><i class="trash alternate outline icon"></i> Delete</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="card">
+                <div class="content">
+                    <button class="ui blue labeled icon button" ng-click="showAddAuthorModal()"><i class="plus circle icon"></i> Add author</button>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="ui modal" id="modal-author">
+        <i class="close icon"></i>
+        <div class="header">
+            {{ authorModalTitle }}
+        </div>
+        <div class="content">
+            <div class="description">
+                <input type="hidden" name="authorId" ng-model="form.author.id" />
+                <form class="ui form">
```

Comment:
For some reason, pressing "enter" here in the input boxes doesn't submit the form. Not a big deal, but I though you might want to know.

Reply:
I was wondering if I would be the only one bothered by this one :). I'll fix it.